scss/styles.scss

```diff
@@ -52,6 +52,23 @@
   }
 }
 
+.image-container {
+  position: relative;
+
+  &::before {
+    background: url("/src/img/badge.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    content: "";
+    height: 100px;
+    position: absolute;
+    right: calc(-94.53px/2);
```

Comment:
i would use percentages here. just to be sure it doesnt break if the width of the image changes